HealthcareSystem/PatientWebApp/Controllers/PatientController.cs

```diff
@@ -175,5 +175,20 @@ public IActionResult GetMaliciousPatients()
             }
         }
 
+
+        [HttpPut("blocked/{jmbg}")]
+        public ActionResult CancelExamination(string jmbg)
```

Comment:
Also, I just noticed now, why is this called CancelExamination? I assume it's an accident, change to BlockPatient or something.

Reply:
Fixed in be98915f313550c2f0c40f3f02babbc905a76b36.